src/assets/scripts/client/constants/aircraftConstants.js

```diff
@@ -132,6 +150,16 @@ export const PERFORMANCE = {
      */
     LANDING_ASSIGNED_SPEED_DISTANCE_NM: 5,
 
+    /**
+     * Maximum vertical distance between the aircraft and the glidepath to
+     * consider the aircraft to be "established on the localizer"
```

Comment:
on the GLIDESLOPE, dummy

Reply:
is that `dummy` pointed at you, as in _you're the dummy_?

Reply:
I am the dummy, yes :smiley: